.circleci/config.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           keys:
             - v7-dependency-cache-{{ checksum "yarn.lock" }}
             - v7-dependency-cache-
+      - run: yarn compile --scope "@blueprintjs/tslint*"
```

Comment:
since circle runs `yarn compile` above with no scope, shouldn't this already be done?

Reply:
`lint` and `compile` run in _parallel_ so they don't affect each other 😢